repos/spack_repo/builtin/build_systems/oneapi.py

```diff
@@ -219,10 +220,36 @@ def openmp_libs(self):
             )
 
         # query the compiler for the library path
+<<<<<<< HEAD:var/spack/repos/spack_repo/builtin/build_systems/oneapi.py
+<<<<<<< HEAD:var/spack/repos/spack_repo/builtin/build_systems/oneapi.py
+        with CompilerPropertyDetector(self["c"].spec).compiler_environment():
+            omp_lib_path = Executable(self["c"].cc)(
+                "--print-file-name", f"{libname}.{dso_suffix}", output=str
+            ).strip()
+||||||| parent of 15700f5e99 (Workaround for broken oneapi library search):lib/spack/spack/build_systems/oneapi.py
```

Comment:
The conflicts need to be resolved.

Reply:
Oof, sorry and thanks for spotting that! Fixed.